src/wx/opts.cpp

```diff
@@ -219,6 +219,7 @@ opt_desc opts[] = {
 
     /// GBA
     STROPT("GBA/BiosFile", "", wxTRANSLATE("BIOS file to use, if enabled"), gopts.gba_bios),
+    BOOLOPT("GBA/LCDFilter", "GBALcdFilter", wxTRANSLATE("Apply LCD filter, if enabled"), gbaLcdFilter),
```

Comment:
might as well make this a global option instead of just exclusive on GBA since utilUpdateSystemColorMaps() affects gba and gb/gbc colormaps or just give each system its own menu item so one can enable on GBA and disable on GB got e.g. for users who want some effect combinations.

Reply:
It already is global, I just added it exclusively to the GBA menu part. If you toggle it while playing a GB, it will work on the GB game. Would you still want it in both menus partitions?

Reply:
it should be on a global menu if it was intended to be global, not on the GBA menu and apply it globally just do no make sense.

also whatever value is set should apply directly at startup 
https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/src/wx/panel.cpp#L1470

Reply:
I'm going to agree with @negativeExponent on this one, makes more logical sense to do that instead of the gba menu. But that's my 2 cents on the matter.

Reply:
not everyone played on gbasp, which is where this lcd saturation was based upon, causes too much saturation for some titles